src/resources/views/crud/fields/upload_multiple.blade.php

```diff
@@ -201,21 +211,45 @@ function bpFieldInitUploadMultipleElement(element) {
 					}
 		        	// if the file container is empty, remove it
 		        	if ($.trim(container.html())=='') {
-						//$('<input type="hidden" name="'+fieldName+'[]" value="">').insertBefore(fileInput);
 		        		container.remove();
 		        	}
 		        	$("<input type='hidden' class='clear-files' name='clear_"+fieldName+"[]' value='"+$(this).data('filename')+"'>").insertAfter(fileInput);
 		        });
 
 		        fileInput.change(function() {
-	                inputLabel.html("{{trans('backpack::crud.upload_multiple_files_selected')}}");
 					let selectedFiles = [];
 
 					Array.from($(this)[0].files).forEach(file => {
 						selectedFiles.push({name: file.name, type: file.type})
 					});
 
 					element.find('input').first().val(JSON.stringify(selectedFiles)).trigger('change');
+
+					// create a bunch of span elements with the selected files names to display in the label
+					let files = '';
+					selectedFiles.forEach(file => {
+						files += '<span class="badge text-bg-secondary badge-primary">'+file.name+'</span> ';
```

Comment:
```suggestion
						files += '<span class="badge badge-outline text-secondary">'+file.name+'</span> ';
```

The dark badge screams too much for attention for my taste. Imho the badge shouldn't be in-your-face, it's not an "important" thing on the screen, if you think in terms of the entire form. So I would rather go for a more muted badge. That being said, figuring out what classes to use will be tricky, because it has to look ok on all 3 themes 🤦‍♂️ And it's possible some themes don't have the classes.

If we can find a more "muted" badge that works OK for all themes, fine.
If not, we can 100% go with the one you chose here. Works good enough on most themes imho.

Reply:
We are on the `"If not, we can 100% go with the one you chose here. Works good enough on most themes imho."` stage. 

I've tested a bunch of configurations that wouldn't require changes in themes. (theme are a mess). This is the only combination I could find that look "okyish" in all the themes. 

Major of the problems is not the fact that themes don't have such classes, is the inconsistency between the same class across different themes. 

So without messing with themes, this is the solution 🤷 and IMO it looks "okay".